capture.py

```diff
@@ -416,6 +421,17 @@ def _isolated_nodes(nodes, panel):
             cmds.isolateSelect(panel, addDagObject=obj)
     yield
 
+@contextlib.contextmanager
+def _maintained_time():
+    """Context manager for preserving (resetting) the time after the context"""
+    from maya import cmds
```

Comment:
I'm looking at all other functions, they all import cmds on their own. Don't remember why I did that originally, but it looks like a bad habit. :/